src/main/java/barcus/tasklist/TaskList.java

```diff
@@ -0,0 +1,119 @@
+package barcus.tasklist;
+
+import barcus.task.Task;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class to encapsulate the list of tasks
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+    private int length;
```

Comment:
Is the private field "length" necessary since we can call this.tasks.size() in its place?